### PR TITLE
Define Voltage, Current

### PR DESCRIPTION
**Pull request by ven-k**
- Define & export `Voltage`, `Current` signals in Electrical module
- Adds tests to verify shape + signal way of defining analog sources

This finishes up the decoupling of Analog devices from shapes (from deprecated #18)

(While here, comments `Short`

### CODE REVIEW

**Comment by ven-k** [comment]
@ChrisRackauckas can you approve the workflow runs?

**Comment by ven-k** [comment]
@ChrisRackauckas can you approve the workflow runs one more time?

Also, now this PR is ready.

**Comment by ValentinKaisermayer** [comment]
Can you please add the smooth signal definitions to `Blocks` and add `Smooth` versions of the sources?

File: src/Electrical/Analog/sources.jl [hunk 1]
```diff
@@ -1,744 +1,33 @@
-# Define and register smooth functions
-_cos_wave(t, f, A, st, ϕ) = A*cos(2*π*f*(t - st) + ϕ)
-_sin_wave(t, f, A, st, ϕ) = A*sin(2*π*f*(t - st) + ϕ)
-_damped_sine_wave(t, f, A, st, ϕ, d) = exp((st-t)*d)*A*sin(2*π*f*(t-st) + ϕ)
-_ramp(t, δ, st, et, h) = h/(et-st)*(_xH(t, δ, st) - _xH(t, δ, et))
-_square_wave(t, δ, f, A, st) = A*2atan(sin(2π*(t-st)*f)/δ)/π
-_step(t, δ, h, a) = h*(atan((t-a)/δ)/π + 0.5)
-_triangular_wave(t, δ, f, A, st) = A*(1-2acos((1 - δ)sin(2π*(t-st)*f))/π)
-_xH(t, δ, tₒ) = (t-tₒ)*(1+((t-tₒ)/sqrt((t-tₒ)^2+δ^2)))/2
```
Comment (ValentinKaisermayer) [inline src/Electrical/Analog/sources.jl:9]: Don't throw these away.

Comment (ven-k) [reply src/Electrical/Analog/sources.jl:9]: I want to add them to Blocks. (But with a separate PR)

**Comment by ven-k** [comment]
Fixed Ramp test in `test/analog.jl` (along with `saveat=0.01`)

fixed-ramp

**Comment by ven-k** [comment]
Now that #59 is complete, once that is merged, I'll add tests with Triangular and Square sources + Voltage/Current

File: test/Electrical/analog.jl [hunk 2]
```diff
@@ -106,97 +114,63 @@ end
...
+        @test_nowarn sol = solve(prob, Rodas5())
         @test_nowarn sol = solve(prob, Tsit5())
-        @test_nowarn sol = solve(prob, Rodas4())
 
-        # Plots.plot(sol; vars=[source.v, capacitor.v])
+        # Plots.plot(sol; vars=[voltage.v, capacitor.v])
     end
 end
 
-# RL with different voltage sources
```
Comment (ValentinKaisermayer) [inline test/Electrical/analog.jl:140]: Why remove this test?

File: src/Electrical/Analog/sources.jl [hunk 3]
```diff
@@ -1,744 +1,33 @@
...
 end
 
 """
-```julia
-function SquareCurrent(; name, offset=0.0, amplitude=1.0, frequency=1.0, start_time=0.0)
-```
+    Current(;name)   
 
-A source in which the current through its terminals is a square function of time.
+Acts as current signal.
```
Comment (ValentinKaisermayer) [inline src/Electrical/Analog/sources.jl:21]: Add a description about the convention of a positive current, i.e. which port has inflowing current if `I` is positive.

Comment (ValentinKaisermayer) [reply src/Electrical/Analog/sources.jl:21]: Also here, it is an ideal current source with no internal resistance.

File: src/Electrical/Analog/sources.jl [hunk 4]
```diff
@@ -1,744 +1,33 @@
...
 """
-```julia
-function SineCurrent(; name, offset=0.0, amplitude=1.0, frequency=1.0, start_time=0.0, phase=0.0)
-```
-
-A source in which the current through its terminals is a sine function of time.
-
+    Voltage(;name)   
 
+Acts as voltage signal.
```
[the remark this answers is not in the document]
Comment (ven-k) [reply src/Electrical/Analog/sources.jl:4]:
```
It is an ideal arbitrary Voltage waveform generator with no internal resistance, i.e. can source infinite currents.

# States
- `v(t)`: [`V`]
  The voltage across this component

# Connectors
- `p`
  Positive pin
- `n`
  Negative pin
- `V`
  Input
```

File: test/Electrical/analog.jl (hunk 2 above)
Comment (ValentinKaisermayer) [reply test/Electrical/analog.jl:140]: No it's ok I think, as long as the RL test is there.

**Review by ValentinKaisermayer — APPROVED** [review]

**Pull request merged**